src/lib.rs

```diff
@@ -51,6 +51,7 @@ pub async fn run_trin(
         listen_port: trin_config.discovery_port,
         no_stun: trin_config.no_stun,
         bootnodes: trin_config.bootnodes.clone(),
+        enr_file_location: Some(node_data_dir.clone()),
```

Comment:
Hey @KolbyML I think this is still somewhat confusing. The name `enr_file_location` implies that the variable is a path to the enr file itself.  But it's actually representing an `Option` for the data dir. A trin node data dir should never be an option, since we always have a data dir. We're also joining the file name to `enr_file_location` inside `discovery.rs`, which is also confusing (eg joining the enr file location to itself).

In this case, I think the simplest way forward is to replace `enr_file_location` with `trin_data_dir` in `PortalnetConfig`

Reply:
Ok that is fine. I will need to append ''nodeid/trin.enr'' to the path I write to then in Discovery::new. Also that sounds about right on it not being an option.

Reply:
Age Manning gaving me some insight on  my PR on sigp/enr so I wanted to do a few changes as well on how it worked!